src/repository/documents.rs

```diff
@@ -20,13 +20,13 @@ pub enum DocumentGetError {
 #[async_trait]
 pub trait DocumentRepository {
     async fn get(&self, projcet_id: i32) -> Result<Vec<Document>, DocumentGetError>;
```

Comment:
Typo in `project_id`, also I think this method should also take in the `user_id`, which should also be in the WHERE clause of the SQL statement.

Reply:
That's true ( also other queries for document/resources needs to take both project_id and user_id to be correct).
In addition to that, we need additional table and some endpoints to store entries for project accesses for non owner users, and document/resources implementation depends on it too. I'd like to move commiting those features in another PR as the current one is quite big already